package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "react-chartjs-2",
-  "version": "2.10.0",
+  "version": "3.0.0",
```

Comment:
Discussion: Should this be 3.0.0? That will conflict with chart.js 3. I would  recomment for now doing a 2.10.0-react-16 or similar. Thoughts?

Reply:
That is a good point. I moved it up to `3.0.0` because it is technically a breaking change, but I can see `2.10.0-react16` being acceptable. We can mostly keep this track of work as pre-releases up until chart.js 3 support is added.

Reply:
Perhaps 3.0.0-alpha is acceptable?